src/unstract/sdk/ocr.py

```diff
@@ -0,0 +1,41 @@
+from abc import ABCMeta
+
+from unstract.adapters.constants import Common
+from unstract.adapters.ocr import adapters
+from unstract.adapters.ocr.ocr_adapter import OCRAdapter
+
+from unstract.sdk.adapters import ToolAdapter
+from unstract.sdk.constants import LogLevel
+from unstract.sdk.tool.base import BaseTool
+
+
+class OCR(metaclass=ABCMeta):
+    def __init__(self, tool: BaseTool):
+        self.tool = tool
+        self.ocr_adapters = adapters
+
+    # @abstractmethod
+    # def convert_to_text(self, input_file: str, basic_convert: bool = False):
+    #     pass
+
+    def get_ocr(self, adapter_instance_id: str) -> OCRAdapter:
```

Comment:
@gaya3-zipstack this function returns `None` as well. Change the return type to `Optional[OCRAdapter]`
Was this not caught by mypy?

Reply:
Yeah wondering why! Same with X2Text. also. Thanks for pointing out. Fixing both

Reply:
Resolved